spring-integration-ftp/src/test/java/org/springframework/integration/ftp/outbound/FtpServerOutboundTests.java

```diff
@@ -182,6 +189,19 @@ public void testInt2866LocalDirectoryExpressionGET() {
 	}
 
 	@Test
+	public void testGetWithRemove() {
+		String dir = "ftpSource/";
+		this.getGw.setOption(Option.DELETE);
```

Comment:
Ah! `AFTER_EACH_TEST_METHOD` does the trick. 😄 

Never mind. Sorry for the noise. 
Merging as is...